scientist-softserv/atla-hyku
# Types controlled vocab

# Story
Add the controlled vocabulary for Types
Make resource type map to the hyrax mapping instead of dc type
Implement this across all 3 worktypes

- #48 

# Expected Behavior Before Changes
Resource type was mapped to `dc:type`
Types did not exist

# Expected Behavior After Changes
- [ ] Types will be a controlled vocab on the edit work form in all three work types, and save successfully
- [ ] Resource Type will be a controlled vocab on the edit work form in all three work types, and save successfully

# Screenshots / Video
<details>
<summary>Edit work form</summary>
<img width="890" alt="image" src="https://github.com/scientist-softserv/atla-hyku/assets/73361970/4337c512-59ad-40ef-8b79-721ad364e9e9">


</details>
<details>
<summary>Saved data on work-show page</summary>
<img width="865" alt="image" src="https://github.com/scientist-softserv/atla-hyku/assets/73361970/6dad6e18-78ae-46a9-a5de-78c0ea537f0b">

</details>

# Testing instructions
- check that you are able to save works with Types & Resource Types in all 3 worktypes- ETD, Paper or Report, and Generic.
